examples/sending-files/server.py

```diff
@@ -35,63 +51,68 @@ async def file_request(
         secure=settings.minio.secure,
     )
 
-    found = client.bucket_exists(current_user.name)
-    if not found:
-        client.make_bucket(current_user.name)
-    else:
-        logger.debug(f"Bucket {current_user.name} already exists")
+    if msg.confirmation:
+        try:
+            response = client.get_object(current_user.name, f"{msg.name}.png")
+
+            image = Image.open(io.BytesIO(response.data))
+
+            image_cropped = image.crop((155, 65, 360, 270))
+            save_image(
+                client,
+                image_cropped,
+                settings.minio.temp_path,
+                f"{msg.name}-cropped.png",
+                current_user.name,
+            )
+
+            image_greyscale = image.convert("L")
+            save_image(
+                client,
+                image_greyscale,
+                settings.minio.temp_path,
+                f"{msg.name}-BW.png",
+                current_user.name,
+            )
+
+            res = ImageModifiedResponse()
+
+            res.name = msg.name
+            res.image_cropped_URL = client.get_presigned_url(
```

Comment:
why are you GETting using presigned URLs? You have a privileged connection with the server, you should be able to fetch it directly, no?